ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/BonsaiPersistedWorldState.java

```diff
@@ -116,7 +126,10 @@ protected Hash calculateRootHash(
     // next walk the account trie
     final StoredMerklePatriciaTrie<Bytes, Bytes> accountTrie =
         new StoredMerklePatriciaTrie<>(
-            this::getAccountStateTrieNode,
+            (location, hash) ->
```

Comment:
it appears that `Optional<Bytes> getAccountStateTrieNode()` is no longer used anywhere